compiler-core/src/parse/snapshots/gleam_core__parse__tests__function_invalid_signature.snap

```diff
@@ -0,0 +1,13 @@
+---
+source: compiler-core/src/parse/tests.rs
+expression: "\nfn f(a, \"b\") -> String {\n    a <> b\n}\n"
+---
+error: Syntax error
+  ┌─ /src/parse/error.gleam:2:9
+  │
+2 │ fn f(a, "b") -> String {
+  │         ^^^ I was not expecting this
+
+Found "b", expected one of: 
```

Comment:
For any token with content can you say "a string", "a float", "an int" etc rather than display the content a second time in the error. Thank you